extension-manifest-v3/src/pages/panel/views/tracker-details.js

```diff
@@ -42,7 +63,19 @@ export default {
     stats.trackers.find((t) => t.id === trackerId),
   wtmUrl: ({ tracker }) =>
     `https://www.whotracks.me/trackers/${tracker.id}.html`,
-  content: ({ tracker, wtmUrl }) => html`
+  requestsObserved: ({ tracker }) =>
```

Comment:
This should be moved to the store model definition - as the `Tracker` have `requestsBlockedCount` and `requestsModifiedCount`. If we need whole objects (not only counts) I would change it in the definition to be `requestsModified` etc...

Reply:
I kept it here as it is the only client and wasn't sure if having it in a model would not cause unnecessary computation.

Reply:
Getters in models are lazy - it means, that they are calculated and cached when they are first time called (for the current instance). It is safe to move it there, and I would keep data and logic separated as much as possible.